### ISSUE TITLE
Newline after <sup> and <em> elements in generated html creates formatting error on display

### ISSUE THREAD

**Issue by hachacha** [BirkbeckCTP/pandoc_plugin#17]
Currently running 44d1fdaa75196e4a9472699bae5bf5b933e16c57

This is it after rendered fully, note the spaces and weird underlines

![image](https://user-images.githubusercontent.com/1020384/86625398-04368100-bf93-11ea-8bc3-44ebccf987d3.png)

When looking at the html file it appears like this for both `<sup>` and `<em>` elements:

![image](https://user-images.githubusercontent.com/1020384/86618580-2d511480-bf87-11ea-97f0-268906d392b8.png)

After removing the new lines from the html galley file directly, the page renders properly with no weird spacing around the sup or emphasis elements.

I tried to debug this to figure out where it's coming from. I checked the output from pandoc because that would make sense since it's these elements specifically giving issue, but don't see anything in stdout:
```
From Isidore\xe2\x80\x99s <em>Etymologies</em> (12<sup>th</sup> century), original located at the British Library (shelfmark: Royal 12 F. IV, f.135v).</p>\n<p>The idea of the impossibility of life in the southern hemisphere (virtually lying on the back side of traditional T-O maps)<a href="#fn4" class="footnoteRef" id="fnref4"><sup>4</sup></a> derived from the theory of climate zones, first described by Macrobius in the 5<sup>th</sup>
```
I tried various ways of printing out from beautiful soup but that also seems about the same.

so I thought it may be from trying to write to the file but changing the newline attribute and replacing these things in the string  in views.py still does not help
```
with open(output_path, mode="w", encoding="utf-8", newline='') as html_file:
        print(html.replace("\r","").replace('\n',''), file=html_file)
```

any help or guidance with this would be greatly appreciated.

**Comment by hachacha** [comment]
Hi,
@ajrbyers @mauromsl  
Wanted to follow up on this. Has this been an issue for any other journals using this plugin?
I have tried the latest version, and updated pandoc to that latest version as well but am still experiencing this extra space on `<sup>` elements when converting to html. 
Thanks.

**Comment by mauromsl** [comment]
Hi @hachacha, can you share the original .docx with us on discord to take a look?
We haven't come across this problem on any journal yet

**Comment by hachacha** [comment]
I tried just converting the doc on the command line on the same server and it output okay. I think the issue may be with beautiful soup somehow...
`pandoc -s ORIGINAL.docx -t html -o foo.html`
![image](https://user-images.githubusercontent.com/1020384/126356553-c171177b-d97e-4494-b4ab-c47eb4b0072d.png)

i'll send the file over on discord thanks!

**Comment by mauromsl** [comment]
closed by birkbeckctp/janeway#4044
